### PR TITLE
Management Updates

### PR DESCRIPTION
**Pull request by wilsonmcdade**
Done:
* Improved Edit Page
    * Thumbnail generation
    * All fields editable on website (not artists though)
    * Feedback viewing
    * Confirmation modal on mural entry deletion
* New Fields on Mural Entries
    * Added a "remarks" field to better distinguish between interpretive labeling and descriptive labeling
    * Added a private notes field to store details that don't need to be public
* New Fields on Images
    * Attribution for photog, differentiating from caption
    * Date taken field
* Added a contact field to the feedback form so we can reach out to people if needed
* Fixed bug where suggestion form thought it was expanded by default
* Added default image for use when S3 file is invalid
* Add field management on edit page
* Add some helper text describing some of the fields displayed on a mural highlight page
* Fixed bugs effecting image ordering
* Moved metadata fields around on mural page
* Add some quick action buttons on the admin panel
* Added tag and artist creation and management on admin panel
* Added tags to mural listings on catalog page
* "Next mural" picker on edit page

### CODE REVIEW

**Comment by MoralCode** [comment]
i see this at least partially addresses #22

File: templates/edit.html [hunk 1]
```diff
@@ -1,20 +1,60 @@
...
                     <label for="caption" class="form-label">Caption</label>
-                    <input value="{{ muralDetails['images'][loop.index-1]['caption'] }}" type="text" class="form-control" id="caption" name="caption" required>
+                    <p class="thin">Write a caption for this image. This should not be a description, as that is covered by the alt text and description fields for the mural. Also include attribution here.</p>
+                    <textarea type="text" class="form-control" id="caption" name="caption">{{ muralDetails['images'][loop.index-1]['caption'] }}</textarea>
                 </div>
                 <div class="mb-3">
                     <label for="alttext" class="form-label">Alt Text</label>
-                    <input value="{{ muralDetails['images'][loop.index-1]['alttext'] }}" type="text" class="form-control" id="alttext" name="alttext" required>
+                    <p class="thin">Write a descriptive caption of this image in particular. Refer to <a href="https://developers.google.com/tech-writing/accessibility/self-study/write-alt-text#write-useful-text">this article</a> on writing good alt text.</p>
```
Comment (humannum14916) [inline templates/edit.html:47]: Maybe reword to "a description of this image" to avoid using "caption" since that refers to the previous field?

File: app.py [hunk 2]
```diff
@@ -517,6 +695,15 @@ def deleteMuralEntry(id):
...
     )
+    db.session.execute(
+        db.delete(MuralTag)
+            .where(MuralTag.mural_id == id)
+    )
+    db.session.execute(
+        db.delete(Feedback)
+            .where(Feedback.mural_id == id)
+    )
+    db.session.query(Mural).filter_by(nextmuralid = id).update({'nextmuralid' : None})
```
[the remark this answers is not in the document]
Comment (wilsonmcdade) [reply app.py:706]: That's a good point. I'll add that

**Review by humannum14916 — APPROVED** [review]
